Rahin_Eugene/02GIT.LOCAL/02.GIT.LOCAL

```diff
@@ -0,0 +1,83 @@
+1.
```

Comment:
Please use git markdown for reports
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax